modules/compliance-custom-scc.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-custom-scc_{context}"]
+= Creating a custom SCC for the Compliance Operator
+
+In some environments, creating a custom Security Context Constraints (SCC) file must be created to ensure the correct permissions are available to the Compliance Operator `api-resource-collector`. 
```

Comment:
"be created" is passive voice, so maybe:

In some environments, you must create a custom Security Context Constraints (SCC) file...